provisionerd/runner/runner.go

```diff
@@ -895,8 +895,9 @@ func (r *Runner) buildWorkspace(ctx context.Context, stage string, req *sdkproto
 			})
 		case *sdkproto.Provision_Response_Complete:
 			if msgType.Complete.Error != "" {
-				r.logger.Info(context.Background(), "provision failed; updating state",
+				r.logger.Warn(context.Background(), "provision failed; updating state",
```

Comment:
note: logging at level ERROR breaks existing unit tests, and I don't want to set `slogtest.IgnoreErrors`.

Reply:
Understandable. It might be the "correct" approach though, if we expect errors to be logged, then setting IgnoreErrors seems like the right course of action.

Reply:
@mafredri Would you agree that anything "ERROR" and above should be mainly for events useful to a system administrator?

Reply:
Could be relevant for e.g. template authors, but generally I doubt coder/provisioner logs (via logger) would be surfaced to anyone but systems admins.